### ISSUE TITLE
"didn't export a Plugin instance"

### ISSUE THREAD

**Issue by jacobrask** [jruchaud/babel-gettext-plugin#1]
Can't get the plugin to work, am I doing something wrong?

```
$ babel --plugins babel-gettext-plugin src/store.js
TypeError: The plugin "babel-gettext-plugin" didn't export a Plugin instance
    at PluginManager.validate (/Users/jacobrask/.nvm/versions/io.js/v2.1.0/lib/node_modules/babel/node_modules/babel-core/lib/transformation/file/plugin-manager.js:164:13)
```

I get the same issue with Node v4.0.0.

Thanks.

**Comment by jruchaud** [comment]
I think, the syntax to declare a plugin for babel is changed. So I will correct.

**Comment by jruchaud** [comment]
Ok it is fixed, you have a new version 1.6.0. The version have been tested on node 4 and Babel 5.8.23.

**Comment by jacobrask** [comment]
Thanks for the quick fix! :)
